core/templates/components/common-layout-directives/navigation-bars/top-navigation-bar.component.ts

```diff
@@ -71,7 +71,20 @@ export class TopNavigationBarComponent implements OnInit, OnDestroy {
   @Input() headerText!: string;
   @Input() subheaderText!: string;
 
-  IMPACT_REPORT_LINK = AppConstants.IMPACT_REPORT_LINK;
+  impactReports = [
+    {
+      link: AppConstants.IMPACT_REPORT_LINK_2023,
+      text: '2023',
```

Comment:
text should be year

Also maybe it's better to have a more general constant in AppConstants and use it both here and in the sidenav. Otherwise you are duplicating information. (The className shouldn't be part of this dict since it's code and not data.)